packages/core/src/utils/IconButton.tsx

```diff
@@ -11,9 +11,20 @@ export interface IconButtonProps extends Omit<HvButtonProps, "icon" | "title"> {
 export const IconButton = ({
   title,
   placement = "top",
+  disabled,
   ...others
-}: IconButtonProps) => (
-  <HvTooltip enterDelay={500} title={title} placement={placement}>
-    <HvButton icon {...others} />
-  </HvTooltip>
-);
+}: IconButtonProps) => {
+  const button = (
+    <HvButton icon disabled={disabled} aria-label={title} {...others} />
+  );
+
+  if (disabled) {
+    return button;
+  }
```

Comment:
I think we should "restore" the disabled button + tooltip behaviour.
Previously we were doing something like:
```tsx
<HvTooltip title="label">
  <div> // <-- focusable, but gets the aria-label
    <HvButton disabled>
```
but we get `aria-label` on the wrong element like this


I think the way to approach this is to just not use `<HvButton disabled={disabled}>`
We could use `<HvButton aria-disabled>` and use this prop to disable the button only visually


What do you think?

Reply:
It would not look great if we need to toggle the "focusable" though. but this probably shouldn't be the case
```tsx
{...(focusable
  ? {"aria-disabled": true}
  : {disabled: true} 
)}
```

We could instead do like MUI and add a `focusableWhenDisabled` prop that toggles this `disabled`/`aria-disabled` and `tabIndex`

Reply:
Oh right, the way I did it the tooltip doesn't appear when the button is disabled 😅 Good idea, I'll look into `focusableWhenDisabled`!

Reply:
Updated ✅